src/models/__tests__/profile.test.ts

```diff
@@ -45,40 +45,54 @@ const aRetrievedProfile: RetrievedProfile = {
   ...aStoredProfile
 };
 
+const aRetrievedProfileWithEnabledReminder: RetrievedProfile = {
+  ...aRetrievedProfile,
+  isReminderEnabled: true
+};
+
 describe("findLastVersionByModelId", () => {
-  it("should resolve to an existing profile", async () => {
-    const containerMock = ({
-      items: {
-        create: jest.fn(),
-        query: jest.fn(() => ({
-          fetchAll: jest.fn(() =>
-            Promise.resolve({
-              resources: [aRetrievedProfile]
-            })
-          )
-        }))
+  it.each`
+    case                                        | retrievedProfile                        | expectedIsReminderEnabled
+    ${"existing profile"}                       | ${aRetrievedProfile}                    | ${false}
+    ${"existing profile with reminder enabled"} | ${aRetrievedProfileWithEnabledReminder} | ${true}
+  `(
+    "should resolve to an $case",
+    async ({ _, retrievedProfile, expectedIsReminderEnabled }) => {
+      const containerMock = ({
+        items: {
+          create: jest.fn(),
+          query: jest.fn(() => ({
+            fetchAll: jest.fn(() =>
+              Promise.resolve({
+                resources: [retrievedProfile]
+              })
+            )
+          }))
+        }
+      } as unknown) as Container;
```

Comment:
What if we add tests on this each for other optional values in profile? More than 2 optional props will require a refactory to keep readable the unit test.
```suggestion
  it.each`
    case                                        | lastAppVersion     | expectedLastAppVersion | isReminderEnabled | expectedIsReminderEnabled
    ${"existing profile"}                       | ${undefined}       | ${"UNKNOWN"}           | ${undefined}      | ${false}
    ${"existing profile with last app version"} | ${aLastAppVersion} | ${aLastAppVersion}     | ${undefined}      | ${false}
    ${"existing profile with reminder enabled"} | ${undefined}       | ${"UNKNOWN"}           | ${true}           | ${true}
    ${"existing profile with all props"}        | ${aLastAppVersion} | ${aLastAppVersion}     | ${true}           | ${true}
  `(
    "should resolve to an $case",
    async ({
      _,
      lastAppVersion,
      expectedLastAppVersion,
      isReminderEnabled,
      expectedIsReminderEnabled
    }) => {
      const containerMock = ({
        items: {
          create: jest.fn(),
          query: jest.fn(() => ({
            fetchAll: jest.fn(() =>
              Promise.resolve({
                resources: [
                  {
                    ...aRetrievedProfile,
                    lastAppVersion,
                    isReminderEnabled
                  }
                ]
              })
            )
          }))
        }
      } as unknown) as Container;
```

Reply:
Nice, did it in https://github.com/pagopa/io-functions-commons/pull/290/commits/89882489685e6da2f5dea2bae67b335f4963e9d4